srcts/src/main.ts

```diff
@@ -4051,8 +4051,10 @@ function main(): void {
             const newStyle = $("<style>").attr("id", id).html(xhr.responseText);
 
             $head.append(newStyle);
-            setTimeout(() => oldStyle.remove(), 500);
-            setTimeout(() => removeSheet(oldSheet), 500);
+
+            oldStyle.remove();
+            removeSheet(oldSheet);
+            sendImageSize();
```

Comment:
Would be good to have a comment here about this working because (as far as we know), in this case, styles are applied syncronously